WordPress/Classes/ViewRelated/Post/WPLegacyEditPostViewController.m

```diff
@@ -390,7 +390,7 @@ - (void)cancelEditing
     }
 
     UIActionSheet *actionSheet;
-    if (![self.post.original.status isEqualToString:@"draft"] && self.editMode != EditPostViewControllerModeNewPost) {
+    if (![self.post.original.status isEqualToString:PostStatusDraft] && self.editMode != EditPostViewControllerModeNewPost) {
```

Comment:
This is something @diegoreymendez and I have talked about and (I believe) there is a follow up PR to this one that does what your asking.